### ISSUE TITLE
[Azure Toolkit][A11yMAS][Accessibility][Inspect][MAS4.2.1][Eclipse][Wipro]: Inappropriate control type for the Table in "Your Subscriptions dialog"

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1391]
**Repro Steps:**

1. Open and Start Eclipse.
2. Navigate to Tools menu-> "Azure"-> "Azure Sign In"
3. In "Azure Sign in" dialog, tab to and select "Automated" radio button.
4. Tab to and select "Browse" button.
5. In "Create Authentication Files" dialog , select the required subscriptions and select "Start" button.
6. In "Service Principal Creation Status dialog", tab to and select "OK" button.
7. In "Your Subscriptions" dialog , tab to table and observe the control type using inspect.

**Actual Result:**
Control type for table is list.
Narrator table commands such as "Caps+F3", "Caps+F8" are working fine.

**Narrator :**
Tab
CIS_BVT_Helsinki, <say-as interpret-as="spell">1</say-as> of 12, selected, 

**NVDA:**
list
CIS_BVT_Helsinki  check box  selected  not checked  Subscription ID: 0bababbd-f605-4b9e-889e-84cc5b966025

**Expected Result:**
Control type should be Table.
Narrator and NVDA should announce as
Table
ASR Canary Test Subscription 4, <say-as interpret-as="spell">2</say-as> of 12, selected, 
CIS_BVT_Helsinki, <say-as interpret-as="spell">1</say-as> of 12, selected, 

**Platform:**
Eclipse Version:
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
OSBuild Version: Version 1709(Os Build 16299.192)
![your subscription-table-not defined-inspect](https://user-images.githubusercontent.com/22956130/35682738-c3f1cdc2-0787-11e8-8823-7f90033471f4.jpg)
![your subscription-table-not defined-narrator](https://user-images.githubusercontent.com/22956130/35682742-c6e213e8-0787-11e8-807a-d35153a65e0f.jpg)
![your subscriptions-table-nvda](https://user-images.githubusercontent.com/22956130/35682743-c94e0902-0787-11e8-98c9-5c6131430418.jpg)

**Comment by Eskibear** [comment]
duplicated with #1405

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749846
